README.md

```diff
@@ -11,9 +11,9 @@
 - CSS
 - JavaScript
 
-## Live Demo (if available)
+## Live Demo 
```

Comment:
- [ ] Please, need to provide a recorded video presenting the features of the project.